src/components/Avatar/Avatar.tsx

```diff
@@ -135,16 +125,13 @@ const AvatarStatusItems: FC<BaseAvatarProps> = React.forwardRef(
               key={position}
               ref={ref}
               style={{
```

Comment:
Should these be moved into the css module?

Reply:
Done